BlueShift-iOS-SDK/BlueShiftAppDelegate.m

```diff
@@ -908,7 +909,7 @@ - (void)handleActionWithIdentifier: (NSString *)identifier forRemoteNotification
     NSDictionary *pushDetailsDictionary = nil;
     pushDetailsDictionary = notification;
     self.userInfo = notification;
-    if ([notification[kNotificationTypeIdentifierKey] isEqualToString:kNotificationTypeActionable]) {
+    if (notification[kNotificationActions]) {
```

Comment:
should we also check the count here? dashboard might send us empty array

Reply:
The for loop will ignore it but I have added the condition.